### PR TITLE
[native] add README.md for tensor query native app

### PR DESCRIPTION
**Pull request by gichan-jang**
add README.md for tensor query native app

Signed-off-by: Gichan Jang <gichan2.jang@samsung.com>

Self evaluation:
1. Build test: [ *]Passed [ ]Failed [ ]Skipped
2. Run test: [ *]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: native/example_query_object_detection/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: query object detection
+...
+
+## Ubuntu Native NNStreamer Application Example - Objectd Detection using tensor query
```
Comment (jaeyun-jung) [inline native/example_query_object_detection/README.md:5]: fix typo Objectd

Comment (gichan-jang) [reply native/example_query_object_detection/README.md]: Updatd!

File: native/example_query_object_detection/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: query object detection
+...
+
+## Ubuntu Native NNStreamer Application Example - Object Detection using tensor query
+### Introduction
+The client sends the video to the server, the server performs object detection(which requires high-performance work) and send the results to the client.
```
Comment (jaeyun-jung) [inline native/example_query_object_detection/README.md:7]: The client sends a video frame to the server, then the server performs object detection (which requires high-performance work) and sends the result to the client.

File: native/example_query_object_detection/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+#### How to Run
+This example requires specific tflite model and label data.
```
Comment (jaeyun-jung) [inline native/example_query_object_detection/README.md:10]: let's use tensorflow-lite (tflite > tensorflow-lite) :)

File: native/example_query_object_detection/README.md (hunk 2 above)
Comment (gichan-jang) [reply native/example_query_object_detection/README.md]: Thanks, Updated.

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**